dataforge-distributed/src/jvmMain/kotlin/space/kscience/dataforge/distributed/RemotePlugin.kt

```diff
@@ -0,0 +1,36 @@
+package space.kscience.dataforge.distributed
+
+import io.lambdarpc.utils.Endpoint
+import space.kscience.dataforge.context.AbstractPlugin
+import space.kscience.dataforge.context.Plugin
+import space.kscience.dataforge.context.PluginFactory
+import space.kscience.dataforge.context.PluginTag
+import space.kscience.dataforge.names.Name
+import space.kscience.dataforge.workspace.SerializableResultTask
+import space.kscience.dataforge.workspace.Task
+
+/**
+ * Plugin that purpose is to communicate with remote plugins.
+ * @param plugin A remote plugin to be used.
+ * @param endpoint Endpoint of the remote plugin.
+ */
+public class RemotePlugin<P : Plugin>(private val plugin: P, private val endpoint: String) : AbstractPlugin() {
```

Comment:
I still do not understand why do we need wrappers for plaguns. Plagins are only a way to load tasks into workspace. Ther should be a wrapper for a task, not a plugin.

Reply:
But where should endpoints for the tasks be provided? Seems that task declaration is not good enough place. Should I use Meta for this?